modules/sql-engine/src/integrationTest/sql/types/timestamp/test_timestamp_ms.test

```diff
@@ -1,7 +1,6 @@
 # name: test/sql/types/timestamp/test_timestamp_ms.test
 # description: Test milliseconds with timestamp
 # group: [timestamp]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-19804
```

Comment:
plz move this "Ignore" into correct place, where such problem already present.
here : test_approximate_distinct_count.test_ignore

Reply:
the issue has been gone. The mentioned test fails for another reason and it should be investigated separately and created a separate ticket.

Reply:
Test case fails, because we don't allow implicit casts. In this case, we can file a ticket just to somehow fix the confusing error message.